### PR TITLE
Add date support in `from xlsx`

### PR DESCRIPTION
**Pull request by nils-degroot**
# Description

This PR add date support when using the `open` command on a xlsx file, and the using `from xlsx` on a xlsx file.
 
# User-Facing Changes

Currently dates in xlsx files are read as nulls, after this PR this would be regular dates.

# Tests + Formatting

# After Submitting

### CODE REVIEW

**Comment by fdncred** [comment]
nice add! seems pretty unobtrusive.

File: crates/nu-command/src/formats/from/xlsx.rs [hunk 1]
```diff
@@ -145,6 +146,12 @@ fn from_xlsx(
                             Data::Float(f) => Value::float(*f, head),
                             Data::Int(i) => Value::int(*i, head),
                             Data::Bool(b) => Value::bool(*b, head),
+                            Data::DateTime(d) => d
+                                .as_datetime()
+                                .and_then(|d| FixedOffset::east_opt(0).map(|o| (d, o)))
+                                .map(|(d, o)| DateTime::from_naive_utc_and_offset(d, o))
+                                .map(|d| Value::date(d, head))
```
Comment (sholderbach) [inline crates/nu-command/src/formats/from/xlsx.rs:153]: Excel's date time doesn't carry its own timezone info? (or as part of the metadata?)

Should this be Zulu time or local time?

Comment (nils-degroot) [reply crates/nu-command/src/formats/from/xlsx.rs:153]: By default excel does not accound for timezones, its upto the sheet owner to manage this themselfs. What should be the default timezone, just utc?

Comment (fdncred) [reply crates/nu-command/src/formats/from/xlsx.rs:153]: I'd prefer the local timezone

Comment (nils-degroot) [reply crates/nu-command/src/formats/from/xlsx.rs]: For now I set it to local with a backup to Utc

**Comment by fdncred** [comment]
Thanks

**Pull request merged**